src/utl/src/ScopedTemporaryFile.cpp

```diff
@@ -33,4 +33,19 @@ ScopedTemporaryFile::~ScopedTemporaryFile()
   }
 }
 
+std::string createTmpFileName(const char* filename)
```

Comment:
std::tmpnam or std::tmpfile should be used.

Reply:
I have used original file name plus ISO 8601 style date time for the tmp file name, which I thought is preferable over just unique file name. Anyway this approach will not be used anymore as @oharboe suggested alternative solution which is accepted by other contributors as well:
> 
> 1. delete destination file, if it exists
> 2. write a foo.odb.tmp
> 3. rename foo.odb.tmp to foo.odb